test/bigunit.operations.test.ts

```diff
@@ -183,11 +181,157 @@ describe("BigUnit Class Methods", () => {
       },
     );
 
-    test("should multiply two BigUnit instances, respecting a precision override", () => {
-      const unitA = BigUnit.fromNumber(1000, 10);
-      const unitB = BigUnit.fromNumber(1.001, 10);
-      const result = unitA.mul(unitB, 2);
-      expect(result.toNumber()).toBe(1000.0);
+    test("should allow doing operations with BigUnit without precision", () => {
+      const unitA = BigUnit.from("82.1", 10);
```

Comment:
Better to use numbers I think? `fromDecimalString` is really just a convenience for handling serialised data.
```suggestion
      const unitA = BigUnit.from(82.1, 10);
```